README.md

```diff
@@ -52,6 +52,7 @@ You can download and install a PowerShell package for any of the following platf
 | Arch Linux               |                         |                               | [Instructions][in-archlinux]  |
 | Kali Linux               | [.deb][rl-ubuntu16]     | [.deb][pv-deb]           | [Instructions][in-kali]       |
 | Many Linux distributions | [Snapcraft][rl-snap]    | [Snapcraft][pv-snap]          |                               |
+| Raspberry Pi      | [Pi-Apps](https://github.com/Botspot/pi-apps)   | | [Pi-Apps Download](https://pi-apps.io/install.html)|
```

Comment:
Minor request, please align the vertical bars in the table in case someone is reading this in the console.  If you don't mind, please fix line 53 as well :)